build/workflows/update-example-index/process.rego

```diff
@@ -0,0 +1,62 @@
+package process
+
+import rego.v1
+
+symbols := {"keywords": _keywords, "builtins": _builtins}
+
+_keywords[name] := path if {
+	some p in _pages
+	p[0] == "keywords"
+
+	name := p[1]
+
+	path := concat("/", p)
+}
+
+_builtins[name] := path if {
+	some p in _pages
+	p[0] == "builtins"
+
+	l := count(p)
+
+	l == 2
+
+	name := p[1]
+
+	not count({p |
```

Comment:
`not count` looks fishy... even if it's the `>` being negated. `< 2` instead?

Reply:
Hmm, that doesn't look good... I have fixed to your version now.